scipy/stats/tests/test_stats.py

```diff
@@ -4574,6 +4575,105 @@ def test_kurtosistest_too_few_samples():
     assert_raises(ValueError, stats.kurtosistest, x)
 
 
+# class TestMann_Whitney_U():
```

Comment:
Preserving these until I take another look to see if there's anything more to used.

Reply:
All of it is still tested. If I remember correctly, the definition of 'greater' and 'less' was inconsistent in these versions; they are now tested correctly.